File: MyWallet/Front_MyWallet/src/app/_model/bank.ts
export class Bank {
  id: number;
  walletId: number;
  name: string;
  amount: number;

  public static of(walletId: number, name: string, amount: number) {
    let bank: Bank = new Bank();
    bank.name = name;
    bank.walletId = walletId;
    bank.amount = amount;
    return bank;
  }
}

File: MyWallet/Backend_MyWallet/src/main/java/org/koz/wallet/app/bank/model/Bank.java
package org.koz.wallet.app.bank.model;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.math.BigDecimal;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class Bank {
    private Long id;
    private Long userId;
    private Long walletId;
    private String name;
    private BigDecimal amount;
}
